src/database/versions.ts

```diff
@@ -26,16 +26,14 @@ export const dbVersions = {
   }
 };
 
-export const dbHistory = ['v1.0.0', 'v0.0.1'];
+export const dbHistory = ['v1.0.0', 'v0.0.1'] as const;
```

Comment:
why do you cast here ?

Reply:
This is not a cast but a [const assertion](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions). Basically it changes the type from `string[]` to `['v1.0.0', 'v0.0.1']`. This removes the need for `// @ts-ignore`, not really related to `date-fns` but I figured I'd just fix it since I was working on that file.

Reply:
😍👌